src/main/java/persistence/dialect/DefaultPagingStrategy.java

```diff
@@ -0,0 +1,28 @@
+package persistence.dialect;
+
+public class DefaultPagingStrategy extends PagingStrategy {
+
+    private DefaultPagingStrategy() {
+    }
+
+    public static DefaultPagingStrategy getInstance() {
+        return InstanceHolder.INSTANCE;
+    }
+
+    @Override
+    public String doPaging(final String query, final int offset, final int limit) {
```

Comment:
와우... 페이징까지.. 👍 

```suggestion
    public String doPaging(PageQuery pageQuery) {
```

이런관점은 어때요?
`pageQuery.쿼리뭐야?`
`pageQuery.어디부터시작이야?`
`pageQuery.얼마나불러올까?`
객체에게 물어보면 좋을것 같아서요!

`limit < 0` 이책임또한 `PageQuery`가 책임진다면!? 좋을것 같은 상상해봤어요 😄 
```
 if(limit < 0) {
            throw new PersistenceException("limit 은 0보다 작을 수 없습니다.");
}
```

Reply:
오마이갓! 이 부분 만들때 맘에 뭔가 안들었는데 그 당시 딱 안떠올라서 어물쩍 넘어간던곳인데 이렇게 알려주시다니!! 넘 감사합니다. 속이 시원하네요!

Reply:
시원하시다니 다행입니다 😄 😄 😄 😄